source/_posts/2017-05-19 kanban-method

```diff
@@ -45,30 +64,37 @@ To make kanban work, each employee must follow these rules:
 - [ ] Keep order,
 - [ ] Discuss about tasks,
 - [ ] Don't race for who will do more - focus on teamwork,
-- [ ] When somebody have problem our task is to help together. 
+- [ ] When somebody have problem our task is to help each other.
 
-These rules is only a sample but improve the whole process, there aren't create the artificial stack of tasks for verification, 
-such tasks are mostly inaccurate and return to the left. Good comunication helps, what if some feature is expired and we dont know 
-about it? One sentence in communication can save much of time. 
+These rules are only a sample but improve the whole process. There aren't create
```

Comment:
Don't understand the meaning: `There aren't create the artificial stack...`. Consider to change this sentence to be more clear